sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/DocumentField.java

```diff
@@ -326,6 +323,25 @@ void setValueMap(Map<String, DocumentField> valueMap) {
         this.valueMap = valueMap;
     }
 
+    /**
+     * Get the Currency value.
+     *
+     * @return the valueCurrency value.
+     */
+    public CurrencyValue getValueCurrency() {
+        return valueCurrency;
+    }
+
+    /**
+     * Set the valueCurrency property: Currency value.
+     *
+     * @param valueCurrency the valueCurrency value to set.
+     */
+    public DocumentField setValueCurrency(CurrencyValue valueCurrency) {
```

Comment:
other setters return `void` but this setter returns the object itself and public?